tools/project_gen/project_gen/scripts/temp.yaml

```diff
@@ -0,0 +1,14 @@
+id: "temp_hazard_curve_two_sites_new_weights"
```

Comment:
Unsure if this should be commited.

Reply:
I mean in the sense that we generally create these yaml files as we want create projects. I don't think we'd want to run this specific scenario in the future.